packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -159,6 +159,11 @@ export interface ISemanticColors {
   menuItemBackgroundChecked: string;
 
   /**
+   * The default text color for a selected menu item title.
+   */
+  menuSelectedText: string;
```

Comment:
Can we have this be bodySelected text instead? This is definitely a pattern we'll see outside menus.
Thanks for using the semantic colors!